demo/index.html

```diff
@@ -7,8 +7,7 @@
   <meta content='width=460' name='viewport'>
   <meta content='text/html; charset=utf-8' http-equiv='content-type' />
   <link href='http://fonts.googleapis.com/css?family=Lato:300,400' rel='stylesheet' type='text/css'>
-  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
-  <link rel="import" href="../../polymer/polymer.html">
+  <script src="../bower_components/webcomponentsjs/webcomponents-lite.js"></script>
```

Comment:
This isn't needed -- this element is meant to be a standalone element, so you should test it using `polymer serve`. See `github.com/polymerelements/paper-button` for another element sample.